### DISCUSSION TITLE
Trying to read a  more stable Tilt result

### DISCUSSION THREAD

**Discussion by n0visk** [universam1/iSpindel#611] [Ideas]
Hello developers, I don't know if this update has been proposed before. But I get a little annoyed while I'm trying to do the density calibration of my ispindel. The tilt changes every second and I always get a different result and I need to do an average. I believe that this little difference on the result could deduced the accuracy of ispindel. I had a similar problem with one other project with one temperature sensor and increasing the array and increasing the delay each every reading, I got a more stable accurracy. I belive that if we increase the MEDIANROUNDSMAX and increade the delay time on each interation, we can get a more stable tilt readings. What do you think about it? Thanks

**Comment by thegreatgunbantoad** [comment]
Possibly increasing the delay in ispindel.cpp line 995 may help.  It already takes at least 29 samples from what I can infer, that should be pretty stable.  Currently 2 ms apart so that will update every 68ms (+/- other running overheads) which is certainly faster than you can humanly read it out in AP mode.
